### PR TITLE
[ALLUXIO-2340]Improve error handling in AlluxioMaster constructor

### PR DESCRIPTION
**Pull request by chenzhaomin123**
https://alluxio.atlassian.net/browse/ALLUXIO-2340

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Jenkins, add to whitelist

File: core/server/src/main/java/alluxio/master/AlluxioMaster.java [hunk 1]
```diff
@@ -269,7 +269,7 @@ protected AlluxioMaster() {
         }
       }
     } catch (Exception e) {
-      LOG.error(e.getMessage(), e);
+      //LOG.error(e.getMessage(), e);
```
Comment (gpang) [inline core/server/src/main/java/alluxio/master/AlluxioMaster.java]: Could you delete this line?

Comment (chenzhaomin123) [inline core/server/src/main/java/alluxio/master/AlluxioMaster.java]: already removed

Comment (chenzhaomin123) [inline core/server/src/main/java/alluxio/master/AlluxioMaster.java]: already removed

**Pull request merged**